package.json

```diff
@@ -36,7 +37,8 @@
     "mocha": "^9.1.3",
     "prettier": "^2.5.1",
     "prettier-plugin-solidity": "^1.0.0-beta.19",
-    "solhint": "^3.3.6"
+    "solhint": "^3.3.6",
+    "@poanet/solidity-flattener": "^3.0.7"
```

Comment:
Why do we need this extra dependency when both hardhat and waffle support flattening? https://coinsbench.com/how-to-flatten-a-solidity-file-using-hardhat-924871ab3139 and https://ethereum-waffle.readthedocs.io/en/latest/getting-started.html

Reply:
Ah, didn't show up first in my search. I have switched to using waffle. Could switch to hardhat once that's ready.